liblib/CMakeLists.txt

```diff
@@ -91,8 +91,8 @@ ENDIF()
 IF(Zstd_FOUND)
 SET(TLL_LIBS ${TLL_LIBS} ${Zstd_LIBRARIES})
 ENDIF()
-IF(Bz2_FOUND)
-SET(TLL_LIBS ${TLL_LIBS} ${Bz2_LIBRARIES})
+IF(BZIP2_FOUND)
+SET(TLL_LIBS ${TLL_LIBS} ${BZIP2_LIBRARIES})
```

Comment:
The `FindBzip2` module defines an imported library, so might be better to use that, as it will take care of the header locations too:

```suggestion
SET(TLL_LIBS ${TLL_LIBS} BZip2::BZip2)
```

(and below on the `buildplugin` line)